eng/versioning/external_dependencies.txt

```diff
@@ -88,9 +93,12 @@ org.springframework.boot:spring-boot-starter;2.3.0.RELEASE
 org.springframework.boot:spring-boot;2.3.0.RELEASE
 org.springframework.boot:spring-boot-starter-parent;2.3.0.RELEASE
 org.springframework.data:spring-data-commons;2.3.0.RELEASE
+org.springframework.data:spring-data-redis;2.2.5.RELEASE
+org.springframework.data:spring-data-mongodb;2.2.5.RELEASE
```

Comment:
Are these two spring-data-* versions align with spring-data 2.3.0?

Reply:
Just discussed, we do not change it first.